pkg/helpers/constants.go

```diff
@@ -17,3 +17,7 @@ const (
 	ClusterAuthBYOC            = true
 	SingleAvailabilityZone     = "single"
 )
+
+var (
+	AWSResources = [5]string{"m3.2xlarge", "m4.large", "m4.xlarge", "m5.large", "m5.xlarge"}
```

Comment:
Just a curiosity question but why are we including m3 and m4 instance types in here since we never use them in testing?

Reply:
It is just to randomize, instance size and try to not hit the cache mechanism they implement.

Reply:
:+1: